dlrover/python/elastic_agent/torch/training.py

```diff
@@ -113,6 +113,7 @@ class ElasticLaunchConfig(LaunchConfig):
     node_unit: int = 1
     auto_tunning: bool = False
     exclude_straggler: bool = False
+    npu: bool = False
```

Comment:
`accelerator: str = "gpu"` is better. We may need to support other accelerators.